package.json

```diff
@@ -10,17 +10,22 @@
     "bootstrap": "^5.3.2",
     "bootstrap-icons": "^1.11.1",
     "cborg": "^4.0.3",
+    "protoc": "^1.1.3",
     "ts-protoc-gen": "^0.15.0"
   },
   "devDependencies": {
+    "run-script-os": "^1.1.6",
     "typescript": "^5.2.2",
     "vite": "^4.4.5"
   },
   "scripts": {
     "build": "npm run protoc && tsc && vite build",
     "dev": "vite",
     "preview": "vite preview",
-    "protoc": "protoc --plugin ./node_modules/.bin/protoc-gen-es --es_out src/generated --es_opt target=ts ./bluetooth.proto",
-    "tsc": "tsc"
+    "protoc": "run-script-os",
+    "protoc:default": "protoc --plugin ./node_modules/.bin/protoc-gen-es --es_out src/generated --es_opt target=ts ./bluetooth.proto",
+    "protoc:win32": "protoc --plugin ./node_modules/.bin/protoc-gen-es.cmd --es_out src/generated --es_opt target=ts ./bluetooth.proto",
+    "tsc": "tsc",
+    "postinstall": "npm run protoc"
```

Comment:
the `.proto` files are technically source files so ideally they're included as part of the normal build pipeline, Is there a Rollup plugin for Vite that could be used? 

For now, this is a convenient work around for those getting started